VideoLocker/src/main/java/org/edx/mobile/view/CourseDiscussionPostsThreadFragment.java

```diff
@@ -41,24 +42,34 @@
 public class CourseDiscussionPostsThreadFragment extends CourseDiscussionPostsBaseFragment {
 
     @InjectView(R.id.discussion_posts_filter_spinner)
-    Spinner discussionPostsFilterSpinner;
+    private Spinner discussionPostsFilterSpinner;
 
     @InjectView(R.id.discussion_posts_sort_spinner)
-    Spinner discussionPostsSortSpinner;
+    private Spinner discussionPostsSortSpinner;
 
     @InjectView(R.id.create_new_item_text_view)
     private TextView createNewPostTextView;
 
     @InjectView(R.id.create_new_item_layout)
     private ViewGroup createNewPostLayout;
 
+    @InjectView(R.id.center_message_box)
+    private TextView centerMessageBox;
+
     @InjectExtra(value = Router.EXTRA_DISCUSSION_TOPIC, optional = true)
     private DiscussionTopic discussionTopic;
 
     private DiscussionPostsFilter postsFilter = DiscussionPostsFilter.ALL;
     private DiscussionPostsSort postsSort = DiscussionPostsSort.LAST_ACTIVITY_AT;
 
     private GetThreadListTask getThreadListTask;
+
+    /**
+     * The purpose of this runnable is to defer the fetching of threads for a topic, until we
+     * have fetched the {@link #discussionTopic} object.
+     */
```

Comment:
If this is meant to be a javadoc, then it should start with a description, e.g. you can replace "The purpose of this runnable is to defer" with "Runnable for deferring" or something like that. If it's meant to be just a comment, then don't structure it like a javadoc.